src/index.css

```
@@ -879,6 +935,42 @@ body.mobile-version.android .leaderboard-current-user-sticky {
   transition: opacity 0.4s ease-out, transform 0.4s ease-out;
 }
 
+/* Появление карточек в табе Меню: динамичная анимация, старт сразу при отрисовке */
+@keyframes menuCardEnter {
```

Stylelint (17.3.0)

[error] Expected keyframe name "menuCardEnter" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)